src/System.Numerics.Vectors/src/System/Numerics/Register.tt

```diff
@@ -5,7 +5,7 @@
 <#@ import namespace="System" #>
 <#@ import namespace="System.Linq" #>
 <#@ import namespace="System.Runtime.InteropServices" #>
-<#@ import namespace="System.Diagnostics.Contracts" #>
```

Comment:
Register.cs needs to be regenerated along with this tt file; it's not dead code.